src/acceleration/BaseQNAcceleration.hpp

```diff
@@ -90,42 +90,42 @@ class BaseQNAcceleration : public Acceleration {
   /**
    * @brief Returns all IQN involved data IDs.
    */
-  virtual std::vector<int> getPrimaryDataIDs() const override final
+  std::vector<int> getPrimaryDataIDs() const final
   {
     return _primaryDataIDs;
   }
 
   /**
    * @brief Initializes the acceleration.
    */
-  virtual void initialize(const DataMap &cplData) override final;
+  void initialize(const DataMap &cplData) final;
```

Comment:
Do we really want this? In my opinion, having the override along with final is more explicit.

Reply:
`final` implies `override` which implies `virtual`.

There is no added benefit for keeping the other terms around.

Reply:
It adds visual clutter that contains no extra information.

`final` means `its virtual and cannot be overridden`. Not sure how you can more clear than that.

Reply:
> final means its virtual and cannot be overridden. Not sure how you can more clear than that.

Using `override` is more

Reply:
I disagree but will apply this to the code base to get this moving.